roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -136,3 +136,6 @@ redhat_os_family_extensions:
   - "openEuler"
   - "UnionTech"
   - "UniontechOS"
+
+# If enabled it will allow kubespray to attempt setup even if the distribution is not supported. For unsupported distributions this can lead to unexpected failures in some cases.
+allow_unsupported_distribution_setup: false
```

Comment:
There is a ‘allow_unsupported_distribution_setup’ variable at the https://github.com/kubernetes-sigs/kubespray/pull/9827/files

Should it be declared here again ?

Reply:
Thanks for the reminder, I've put the variable declarations in all.yaml.